src/devtools/client/webconsole/actions/index.ts

```diff
@@ -0,0 +1,35 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export { toggleTimestamps } from "devtools/client/webconsole/reducers/ui";
```

Comment:
:speech_balloon: If you compare vs the old `index.js` file, I pretty heavily rewrote the import/export syntax here. In theory that could cause problems, but I checked for the places that did actually reference this file and updated them.